profiles/adityathakekar.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/19209650/147535896-f3259b35-75cd-4c82-a0fb-202f3c104234.png",
```

This was suggested by @krishguptadev. If you liked it and want to change it, you can click on the `Commit suggestion` button :) Let us know if you need any help!
```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147574750-c64d0af2-4863-4371-8a11-1f96c0531f07.png",
```